terraso_backend/apps/soil_id/models/soil_data_history.py

```diff
@@ -13,13 +13,23 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see https://www.gnu.org/licenses/.
 
+import enum
+
+from django.core.serializers.json import DjangoJSONEncoder
 from django.db import models
 
 from apps.core.models import User
 from apps.core.models.commons import BaseModel
 from apps.project_management.models.sites import Site
 
 
+class JSONEncoder(DjangoJSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, enum.Enum):
+            return obj.value
+        return super().default(obj)
```

Comment:
comment: i don't _think_ this is super specific to the `SoilDataHistory` model, but i'm not sure where to put it or if other models which use it would need to implement it differently so i'm keeping it local for now

Reply:
I think this is fine for now -- if the pattern reoccurs we can extract something more general.